randovania/data/json_data/prime3.txt

```diff
@@ -1575,25 +1843,42 @@ Asset id: 12448241480574438186
 > Dock to Hangar Bay Hall; Heals? False
   * Normal Door to Hangar Bay Hall/Dock to Hangar Bay
   > Dock to Ruined Shrine
-      Trivial
+      After Hangar Bay Power
 
 > Dock to Ruined Shrine; Heals? False
```

Comment:
Add another connection to Samus Ship (mentioned below) with the requirement of Command Visor and After Ceiling Event